packages/webapp/src/components/collection/deckList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Link from "src/components/link"
+import { Deck } from 'src/store/types'
+
+interface DeckCollectionDisplayProps {
+  decks: Deck[]
+  onDeckSelect: (deckID: number) => void
+}
+
+const DeckCollectionDisplay: React.FC<DeckCollectionDisplayProps> = ({ decks, onDeckSelect }) => {
+  return (
+      <div className="w-full flex flex-col items-center p-3">
+        {/* New Deck Button */}
+        <Link 
+          href={"/collection?newDeck=true"} 
+          className="w-full px-4 py-2 mb-2 border rounded-md text-gray-100 bg-purple-900 hover:bg-gray-500 font-bold text-center focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent">
+          New Deck →
+        </Link>

```

Comment:
1. Make this more obviously clickable, maybe a button? This looks like a heading right now.
    I think maybe this should be button-ish, but the listed decks should not, they could be a list of underlined text follow [by one of those little arrows](https://gist.github.com/miguelmota/322c89234d60de578f37d3c6d30f7e41) (or from [react-icons](https://react-icons.github.io/react-icons/)  which we're already importing).
    
2. Make sure there is spacing under it. (1) will probably take care of that, but currently there's no spacing between "New Deck" and the list of decks.